frontend/.devcontainer/Dockerfile

```diff
@@ -0,0 +1,19 @@
+# See here for image contents: https://github.com/microsoft/vscode-dev-containers/tree/v0.194.0/containers/typescript-node/.devcontainer/base.Dockerfile
+
+# [Choice] Node.js version: 16, 14, 12
+ARG VARIANT="14-buster"
+FROM mcr.microsoft.com/vscode/devcontainers/typescript-node:0-${VARIANT}
+
+# [Optional] Uncomment this section to install additional OS packages.
+RUN apt-get update && export DEBIAN_FRONTEND=noninteractive \
+  && apt-get -y install --no-install-recommends git libstdc++6
+
+# [Optional] Uncomment if you want to install an additional version of node using nvm
+ARG EXTRA_NODE_VERSION=v14.19.3
+RUN su node -c "npm config delete prefix"
+RUN su node -c "source /usr/local/share/nvm/nvm.sh && nvm install ${EXTRA_NODE_VERSION}"
+RUN su node -c "npm config set prefix $NVM_DIR/versions/node/${EXTRA_NODE_VERSION}"
+# RUN su node -c "nvm use --delete-prefix ${EXTRA_NODE_VERSION} --silent"
```

Comment:
Don't know how to run the `nvm` command to get rid of the warning.  For some reason `nvm` isn't available at this point...

Reply:
Not sure what you mean?

Reply:
We discussed this in our meeting.  This doesn't work.